cinder/volume/drivers/vmware/vmdk.py

```diff
@@ -2985,11 +2985,42 @@ def migrate_volume(self, context, volume, host):
                 return self._migrate_attached_cross_vc(context, dest_host,
```

Comment:
My point is, that if I call Cinder with a "wrong" target i.e. a target in another vCenter, it will go into `self._migrate_attached_cross_vc()` which does not migrate anything.